src/Sulu/Bundle/ContentBundle/Search/EventListener/ReindexListener.php

```diff
@@ -68,9 +76,24 @@ public function onIndexRebuild(IndexRebuildEvent $event)
 
         $output->writeln('<info>Rebuilding content index</info>');
 
+        $typeMap = $this->baseMetadataFactory->getPhpcrTypeMap();
+
+        $condition = '';
+        foreach ($typeMap as $type) {
+            $phpcrType = $type['phpcr_type'];
+
+            if ($phpcrType !== 'sulu:home' && $phpcrType !== 'sulu:path') {
+                if (!empty($condition)) {
+                    $condition .= ' or ';
+                }
```

Comment:
@danrot +1, also this would not be necessary if https://github.com/jackalope/jackalope-doctrine-dbal/issues/217 were implemented in Jackalope Doctrine Dbal, as `sulu:home` extends `sulu:content`, so selecting from `sulu:content` should be a valid thing to do.